pkg/pillar/evetpm/tpm.go

```diff
@@ -76,6 +77,28 @@ const (
 	// TpmSavedDiskSealingPcrs is the file that holds a copy of PCR values
 	// at the time of generating and sealing the disk key into the TPM.
 	TpmSavedDiskSealingPcrs = types.PersistStatusDir + "/sealingpcrs"
+
+	// TpmEvtLogBinFile is a kernel exposed variable that contains the
+	// TPM event log.
+	TpmEvtLogBinFile = "binary_bios_measurements"
+
+	// TPMEvtLogDir is directory that TPM event log resides in.
+	TPMEvtLogDir = "/hostfs/sys/kernel/security/tpm*"
+
+	// TpmEvtLogSealSuc is files that holds a copy of event log at the time
+	// of generating/sealing the disk key into the TPM.
+	TpmEvtLogSealSuc = types.PersistStatusDir + "/tpmevtlog_success"
+
+	// TpmEvtLogUnsealFail is files that holds a copy of event log at the time EVE
+	// fails to unseal the vault key from TPM.
+	TpmEvtLogUnsealFail = types.PersistStatusDir + "/tpmevtlog_fail"
+
+	// TpmEvtLogSavePattern contains the pattern used to save event log files,
+	// considering the case when multiple tpm devices are present.
+	TpmEvtLogSavePattern = "%s-tpm%d"
+
+	// TpmEvtLogCollectPattern contains the pattern used to collect event log files.
+	TpmEvtLogCollectPattern = "%s-collected"
```

Comment:
Having the printf formatting string be pre-defined isn't clear since it is far from the code using it. 
Better to define functions which return the filename where the fmt string is local.

Reply:
fixed.